examples/gno.land/p/demo/blog/blog.gno

```diff
@@ -27,6 +27,7 @@ func (b Blog) Render(path string) string {
 	switch {
 	case isHome:
 		output := breadcrumb([]string{b.Title})
+		output += "<div class='columns-3'>"
 
 		if b.Posts.Size() == 0 {
```

Comment:
```go
} else {
	output += "<div class='columns-3'>"
	// etc
}
```

Reply:
It's not correct otherwise. (Actually, I would say simply put `output += "<div class='columns-3'>"` after the end of the if block, as that one returns), because otherwise `<div>` is printed even when there are no posts (and no corresponding `</div>`) - like it happens in gnoblog_test.gno.

Reply:
Ha yes, I missed the conditional return. Thank you!